exampleCourse/questions/demo/autograder/python/mustacheProcess/tests/test.mustache.py

```diff
@@ -0,0 +1,27 @@
+from pl_helpers import name, points, not_repeated
+from pl_unit_test import PLTestCaseWithPlot, PLTestCase
+from code_feedback import Feedback
+from functools import wraps
+
+{{#params}}
+class Test(PLTestCaseWithPlot):
```

Comment:
Nit: AFAICT, this example question doesn't deal with plots. Is this intentional? If not, let's use `PLTestCase` instead.

Reply:
Entirely unintentional. I switched this to `PLTestCase` and also deleted unused imports (including `not_repeated`). I haven't tested this yet!